modules/terraform/output.go

```diff
@@ -70,17 +70,15 @@ func OutputListE(t *testing.T, options *Options, key string) ([]string, error) {
 		return nil, err
 	}
 
-	if outputMap, isMap := output.(map[string]interface{}); isMap {
-		return parseListOutputTerraform11OrOlder(outputMap, key)
-	} else if outputList, isList := output.([]interface{}); isList {
-		return parseListOutputTerraform12OrNewer(outputList, key)
+	if outputList, isList := output.([]interface{}); isList {
+		return parseListOutputTerraform(outputList, key)
```

Comment:
Sanity check this is intentional to no longer support TF 0.11 after this release?

Reply:
Yes, based on https://github.com/gruntwork-io/terratest/issues/418#issuecomment-564638173

Will wait until you approve this before I move forward. 

FWIW, I think this is not a huge deal if we implement https://github.com/gruntwork-io/terratest/issues/445. Then, one could use an older version of the `terraform` module that is still compatible with tf11 but newer versions of the other helpers (e.g `aws`, `k8s`, etc).

Reply:
Agreed on dropping TF 11 support. No need to block on #445 at this point.